identity-service/src/relay/txRelay.js

```diff
@@ -242,6 +216,52 @@ const sendTransactionInternal = async (req, web3, txProps, reqBodySHA) => {
   return txReceipt
 }
 
+/**
+ * Rate limit replica set reconfiguration transactions
+ * the available wallets using mod
+ *
+ * A reconfiguration (as opposed to a first time selection) will have an
+ * _oldPrimaryId value of "0"
+ */
+
+const filterReplicaSetUpdates = (decodedABI, senderAddress) => {
+  let isReplicaSetTransaction = false
+  let isFirstReplicaSetConfig = false
+
+  if (decodedABI.name === 'updateReplicaSet') {
+    // TODO remove legacy replica set updates
+    isFirstReplicaSetConfig = decodedABI.params.find(
+      (param) => param.name === '_oldPrimaryId' && param.value === '0'
+    )
+  } else if (decodedABI.name === 'manageEntity') {
```

Comment:
i know this wasn't there before, and not huge deal but - as a safety, should we also check contractName?